pyspark_ai/prompt.py

```diff
@@ -87,7 +87,7 @@
 Observation: 01-01-2006
 Thought: The correct Birthday filter should be '01-01-2006' because it is semantically closest to the keyword.
 I will use the column 'Birthday' to filter the rows where its value is '01-01-2006' and then select the COUNT(`Student`) 
-because COUNT gives me the total number of students.
+because the question asks for the total number of students.
```

Comment:
QQ: does this fix all mismatched all `COUNT` queries?

Reply:
yes! except for maybe a couple, but overall yes

Reply:
This is great!